src/prompts/ddl/deeplink-button.tsx

```diff
@@ -1,4 +1,4 @@
-import { Component, h } from 'preact';
+import { Component, h, Fragment } from 'preact';
```

Comment:
Any reason we import Fragment now, seems unused unless I missed something?

Reply:
Nope, was just residual and now redundant - removed 👍